### PR TITLE
Fix declared python version

### PR DESCRIPTION
**Pull request by eli-schwartz**
setup.py should not allow installation with versions of python that are
*known* not to work.

cf. #23 #25

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -56,18 +56,14 @@
...
-        'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.3',
-        'Programming Language :: Python :: 3.4',
-        'Programming Language :: Python :: 3.5',
         'Topic :: Security',
     ],
 
     packages=find_packages(),
     include_package_data=True,
+    python_requires='~=2.7.13',
```
Comment (ph4r05) [inline setup.py]: I am not sure about this one. 2.7.14 may work also, that may be true also for some 2.7.x versions

Comment (eli-schwartz) [reply setup.py]: Your project README states: "Separate Python 2.7.13 It won't work with lower Python version"
I have not tested it on anything other than my system python2 which is 2.7.14, where it correctly installs, and python3 3.6.2 where it now refuses to install.

https://www.python.org/dev/peps/pep-0440/#version-specifiers defines the format of this version specifier, which declares that it works on any minor version equal to or greater than 2.7.13

EDIT: No, I misread that, it means any version 2.7.* greater than 2.7.13

Comment (eli-schwartz) [reply setup.py]: Fixed and force-pushed. It should accept any hypothetical python 2.8.x now.

**Comment by ph4r05** [comment]
w.r.t. py versions, I just checked py 2.7.10-2.7.14 and all work on travis

https://travis-ci.org/crocs-muni/roca

Moreover I just checked 2.7.6 with pyenv and all tests passed correctly.

**Comment by eli-schwartz** [comment]
Then  you should fix your README instead of unambiguously declaring that the minimum version is 2.7.13 :grin:

This pull request simply converts the instructions in the README into machine-readable python-setuptools keywords.

**Comment by ph4r05** [comment]
The tested version is 2.7.13, if it runs with lower, yes thats possible, I didn't test it till now. 
Py3 support was planned but didn't make it in time so setup.py py3 support is leftover.

**Comment by eli-schwartz** [comment]
So it wasn't initially clear to me that you recommitted these changes as f50c05cb7c2e423db5ce644ff923c4b76a49edca which left me somewhat confused about why this pull request was closed.

In the future if you are going to close a pull request by recommitting the changes yourself, at least mention "fixes #30" in the commit message and Github will close it for you *and* link the commit here so people actually know what happened. See https://help.github.com/articles/closing-issues-using-keywords/

**Comment by ph4r05** [comment]
I changed py versions, >=10, you had >=13.

Yep true, I could have, sorry for that, next time. 

Thanks for contribution.

**Pull request closed without merging**